### PR TITLE
fix-email-templates

### PR DESCRIPTION
**Pull request by jamshidbek7375**
Before

image

After 

image

### CODE REVIEW

File: ExpenseTracker/ExpenseTracker.Application/Stores/WalletStore.cs [hunk 1]
```diff
@@ -144,8 +144,9 @@ await _hubContext.Clients.All
 
             if (user is null)
             {
-                var message = new EmailMessage(userEmail, "there", "Collaboration Invitation", null);
-                _emailService.SendWalletInvitation(message);
+                var inviteSenderName = wallet.Owner.Email;
+                var message = new EmailMessage(userEmail, "", "Collaboration Invitation", null);
```
Comment (Ramadan1011) [inline ExpenseTracker/ExpenseTracker.Application/Stores/WalletStore.cs:148]: I think _username_ should be given to the second parameter of the **EmailMessage** method

**Review by Ramadan1011 — CHANGES_REQUESTED** [review]
Discuss my review with others

File: ExpenseTracker/ExpenseTracker.Application/Stores/WalletStore.cs (hunk 1 above)
Comment (Ramadan1011) [reply ExpenseTracker/ExpenseTracker.Application/Stores/WalletStore.cs:148]: > > I think _username_ should be given to the second parameter of the **EmailMessage** method
> 
> hali ro'yhatdan o'tmagan kishining qanday qilib UserName ni olamiz

Oops! my mistake, I think that the registered user is being invited

**Review by Ramadan1011 — APPROVED** [review]
Great

**Review by Abdulazim1771 — DISMISSED** [review]

**Comment by Boburm1rzo** [comment]
In my opinion, instead of email, it would be better to write a username

**Review by Boburm1rzo — DISMISSED** [review]

**Comment by jamshidbek7375** [comment]
> In my opinion, instead of email, it would be better to write a username

How to write the username of a person who has not yet registered here

**Review by davron41 — CHANGES_REQUESTED** [review]
Instead of user email let's just say "Hi"

**Review by Ramadan1011 — APPROVED** [review]
great

**Review by Boburm1rzo — APPROVED** [review]
👍

**Review by davron41 — APPROVED** [review]
looks good.

**Review by Abdulazim1771 — APPROVED** [review]

**Pull request merged**